sdk/servicebus/azure-servicebus/azure/servicebus/_common/message.py

```diff
@@ -967,6 +969,27 @@ def dead_letter_source(self):
                 pass
         return None
 
+    @property
+    def message_state(self):
+        # type: () -> ServiceBusMessageState
+        """
+        Defaults to Active. Represents the message state of the message. Can be Active, Deferred.
+        or Scheduled.
+
+        :rtype: ~azure.servicebus.ServiceBusMessageState
+        """
+        if self._raw_amqp_message.annotations:
+            try:
+                message_state = self._raw_amqp_message.annotations.get(MESSAGE_STATE_NAME)
+                return (
+                    ServiceBusMessageState.DEFERRED if message_state == 1
+                    else ServiceBusMessageState.SCHEDULED if message_state == 2
+                    else ServiceBusMessageState.ACTIVE
+                )
+            except AttributeError:
+                pass
+        return ServiceBusMessageState.ACTIVE
```

Comment:
I don't think we need the `if` check at the beginning? if self._raw_amqp_message.annotations is None, then `self._raw_amqp_message.annotations.get` triggers `AttributeError`, and we just return `ServiceBusMessageState.ACTIVE` in the except case

```suggestion
        try:
            message_state = self._raw_amqp_message.annotations.get(MESSAGE_STATE_NAME)
            return (
                ServiceBusMessageState.DEFERRED if message_state == 1
                else ServiceBusMessageState.SCHEDULED if message_state == 2
                else ServiceBusMessageState.ACTIVE
            )
        except AttributeError:
            return ServiceBusMessageState.ACTIVE
```